doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -777,20 +735,30 @@ type which is the type of its body.
 A formal presentation of constants and environments is given in
 Section :ref:`typing-rules`.
 
-.. cmd:: Definition @ident := @term
+.. cmd:: {| Definition | Example } @ident_decl @def_body
+   :name: Definition; Example
+
+   .. insertprodn def_body def_body
 
-   This command binds :token:`term` to the name :token:`ident` in the environment,
-   provided that :token:`term` is well-typed.
+   .. prodn::
+      def_body ::= {* @binder } := {? @reduce } @term
+      | {* @binder } : @term := {? @reduce } @term
+      | {* @binder } : @term
```

Comment:
Can we use `type` here and above? This would make the explanation below easier to understand.

```suggestion
      | {* @binder } : @type
```